src/components/fields/FilterOperation.js

```diff
@@ -0,0 +1,196 @@
+import Field from './Field';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import DropdownWidget from '../widgets/Dropdown';
+import TextInput from '../widgets/TextInput';
+import {connectToContainer} from 'lib';
+
+const operators = [
+  {
+    label: 'Inequality',
+    value: 'inequality',
+  },
+  {
+    label: 'Range',
+    value: 'range',
+  },
+  {
+    label: 'Matching values',
+    value: 'set',
+  },
+];
+
+const operations = {
+  inequality: [
+    {value: '<', label: '<'},
+    {value: '<=', label: '≤'},
+    {value: '=', label: '='},
+    {value: '>', label: '>'},
+    {value: '>=', label: '≥'},
+  ],
+  range: [
+    {value: '[]', label: 'Lower ≤ x ≤ Upper'},
+    {value: '()', label: 'Lower < x < Upper'},
+    {value: '[)', label: 'Lower ≤ x < Upper'},
+    {value: '(]', label: 'Lower < x ≤ Upper'},
+    {value: '][', label: 'x ≤ Lower OR x ≥ Upper'},
+    {value: ')(', label: 'x < Lower OR x > Upper'},
+    {value: '](', label: 'x ≤ Lower OR x > Upper'},
+    {value: ')[', label: 'x < Lower OR x ≥ Upper'},
+  ],
+  set: [{value: '{}', label: 'Include'}, {value: '}{', label: 'Exclude'}],
+};
+
+const findOperation = operator => {
+  let op = 'inequality';
+  for (const key in operations) {
+    if (
+      operations.hasOwnProperty(key) &&
+      operations[key].map(o => o.value).indexOf(operator) !== -1
+    ) {
+      op = key;
+      break;
+    }
+  }
+  return op;
+};
+
+class UnconnectedFilterOperation extends Component {
```

Comment:
would you put this component in `components/fields` like for all our other connected components :)